src/angular-app/bellows/core/offline/offline-cache-utils.service.ts

```diff
@@ -29,6 +29,18 @@ export class OfflineCacheUtilsService {
     return this.offlineCache.setObjectsInStore('projects', this.sessionService.projectId(), [obj]);
   }
 
+  getProjectMruEntryData(): Promise<any> {
```

Comment:
Feel free to disregard, but it's pretty inconsistent that we've changed a couple of these method signatures. Ideally they're all the same, right? Or we can just not!

Reply:
I agree, thank you for pointing this out!  Since the plan is to abandon this code, I mostly only interested in fixing code that is broken, instead of refactoring code that is apparently working.

Reply:
I am going to resolve this thread since it is not requesting changes and is not intended to block the PR.